styles/category.scss

```diff
@@ -0,0 +1,65 @@
+main {
+    min-height: calc(100vh - 80px);
+    max-width: 1240px;
+    margin-inline: auto;
+    padding: 2rem 0;
+}
+
+.hero-container {
+    background-image: url("/images/new.png");
+    height: 9rem;
+    background-repeat: repeat;
+    background-size: 80%;
+    border-radius: 0.5rem;
+    background-color: var(--primary);
+    position: relative;
+    color: var(--white);
+    padding: 1rem 3rem;
+    margin-bottom: 2rem;
+
+    .text {
+        position: relative;
+        z-index: 1;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        height: 100%;
+        h1 {
+            margin: 0.7rem 0;
+            line-height: 1;
+        }
+        p {
+            font-size: 0.9rem;
+            width: 50%;
+            line-height: 1.2;
+        }
+    }
+    &::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        border-radius: 0.5rem;
+        z-index: 0;
```

Comment:
I think default value of z index is 0 so what's the point of specifying it here?

Reply:
So here I'm working on 3 different layers. and the `:before` element is on top of something, so that's why it's necessary to mention the index at 0.